src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -163,13 +162,13 @@ int BATT_SMBUS::task_spawn(int argc, char *argv[])
 	return PX4_ERROR;
 }
 
-void BATT_SMBUS::cycle_trampoline(void *arg)
+void BATT_SMBUS::start_cycle(void *arg)
```

Comment:
batt_smbus has start_cycle() with this work as well... that's it.

Reply:
This is a weirder one because of how the module is started. I'll leave it alone for now and do another pass to update the non-trivial ones.